addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -69,7 +69,7 @@ class EnvironmentScService extends Service {
     await super.init();
     const [dbService, environmentAuthzService] = await this.service(['dbService', 'environmentAuthzService']);
     const table = this.settings.get(settingKeys.tableName);
-    this.isAppStreamEnabled = this.settings.get(settingKeys.isAppStreamEnabled) === 'true';
+    this.isAppStreamEnabled = this.settings.optionalBoolean(settingKeys.isAppStreamEnabled, false);
```

Comment:
Can you remove this. I have moved to a different method and added tests, so want to avoid unnecessary merge conflicts.

Reply:
I knew this was coming. I'm keeping an eye on your PR. I'll make an update once you merge :)